packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card/property-type-to-form-data-expected-values.ts

```diff
@@ -41,6 +41,7 @@ export const propertyTypeToFormDataExpectedValues = (
           data: {
             typeId: expectedValue.$ref,
           },
+          ...(parentId ? { parentId } : {}),
```

Comment:
This fix should be made in #1769, just added it here for testing purposes

Reply:
Been added in that PR now